Packages/com.unity.render-pipelines.universal/Shaders/PostProcessing/UberPost.shader

```diff
@@ -40,6 +41,13 @@ Shader "Hidden/Universal Render Pipeline/UberPost"
             #endif
         #endif
 
+
+        #if SUBPASS_INPUT_ATTACHMENT
+            #define urp_cameraColor 0
+            FRAMEBUFFER_INPUT_HALF_MS(urp_cameraColor);
+            int _MSAALevel;
```

Comment:
I'm wondering if moving this to a shader define instead of a dynamic variable would be faster. See CopyDepthPass.hlsl.